common/constants/constants.go

```diff
@@ -21,8 +21,8 @@ const (
 	DYNAMODB string = "dynamodb"
 
 	// Target db for which schema is being generated.
-	TARGET_SPANNER               string = "spanner"
-	TARGET_EXPERIMENTAL_POSTGRES string = "experimental_postgres"
+	TargetSpanner              string = "spanner"
```

Comment:
Why are these constants showing up in the checks and not the others ? All of them are in capitals.

Reply:
It's not the capitalization that the linter is complaining about, but rather the fact that it's using '_' instead of CamelCase